include/tss2/tss2_tcti.h

```diff
@@ -83,6 +83,8 @@ typedef void TSS2_TCTI_POLL_HANDLE;
     ((TSS2_TCTI_CONTEXT_COMMON_V1*)tctiContext)->setLocality
 #define TSS2_TCTI_MAKE_STICKY(tctiContext) \
     ((TSS2_TCTI_CONTEXT_COMMON_V2*)tctiContext)->makeSticky
+#define TSS2_TCTI_RESET(tctiContext) \
```

Comment:
Sorry, but this is TCG-spec.
We cannot just add things in here, esp if they are very simulator specific rather than general-purpose-TPM.

We can add a `Tss2_Tcti_swtpm_Reset()` to tss2_tcti_swtpm.h for this.

Reply:
Oh ok. That's what I suspected. Do you mind if the tests are not TCTI-agnostic? If so, I'll replace mssim-specific commands by swtpm-specific commands.

Reply:
If it's unit test, I don't care, cause they are tcti-specific anyways.
If it's integration tests, I do care, but I cannot remember an int-test to use this anyways (since it was not exported by the .map file)...

Reply:
I'm talking about integration tests. Previously, `tpmclient` and `sapi_command_cancel` call the mssim-specific function `tcti_platform_command()`.
`tpmclient` resets the TPM and `sapi_command_cancel` cancels commands (without using the dedicated TCTI function) and is IMO anyway obsolete.